camviewer_ui_impl.py

```diff
@@ -2230,14 +2255,59 @@ def setCalibPV(self, pvname):
     def onSpecific(self, button):
         pass
 
-    def onOpenEvr(self):
-        iCamera = self.ui.comboBoxCamera.currentIndex()
-        if self.lEvrList[iCamera] != "None":
-            print(
-                "Open Evr %s for camera [%d] %s..."
-                % (self.lEvrList[iCamera], iCamera, self.lCameraList[iCamera])
+    def new_launch_gui_script(self, exc: Exception | None = None):
```

Comment:
Yeah, this is a judgement call for sure. I ultimately decided that having it copied once was easier than formulating the partials because I was thinking about some old article I read about practical coding that insisted that copying once was ok and copying twice was the greatest sin. In retrospect I don't know how much I agree with that article.

Some of these camviewer PRs are a bit rough in that I didn't put in quite as much care as I would with one of our Python libraries, mostly because I'm trying to put camviewer to bed to work on the replacement app. I should probably slow down a bit.

Reply:
I think there's a reasonable middle ground to be found between writing verbose/readable code and playing code golf.  

If camviewer 2.0 is on the horizon, I think it's more than reasonable to button up these quick fixes and move on.

Reply:
Ok, I'll merge this one and spend my attention on the more topical things you both pointed out in the image-saving PR.

If we need to change these again I'll refactor at that time.